Src/Localization/English/Main.html

```diff
@@ -2,48 +2,47 @@
 <html><head>
 
 
-
-
-
-
-
-
+	
+	
+	
+	
+	
   <meta content="text/html; charset=ISO-8859-1" http-equiv="content-type"><title>Open-Shell</title>
   <style type="text/css">
 h1 {
-  color: #0070c0;
-  font-family: "Times New Roman",Times,serif;
+  color: #0078d4;
+  font-family: "Segoe UI",Tahoma,sans-serif;
+  font-weight: 600;
+}
+h2 {
+  color: #005a9e;
+  font-family: "Segoe UI",Tahoma,sans-serif;
+  font-weight: 600;
 }
 body {
-  font-family: Arial,Helvetica,sans-serif;
+  font-family: "Segoe UI",Tahoma,sans-serif;
 }
 
   </style></head><body>
-<h1><a target="_blank" href="http://www.classicshell.net/"><img src="images/OpenShell.png" style="border: 0px solid ; width: 64px; height: 64px;" alt="Open-Shell website" title="Open-Shell website"></a>&nbsp; Open-Shell</h1>
-<span style="font-style: italic; color: rgb(0, 112, 192); font-weight: bold;">Version 4.3.1 &#8211; general release</span><br><br>
-<h1>What is Open-Shell?</h1>
-<strong style="color: rgb(79, 129, 189);">Open-Shell&#8482;</strong> is a collection of usability enhancement for Windows. It 
-has a customizable Start menu and Start button, it adds a 
-toolbar for Windows Explorer and supports a variety of smaller features.<br>
-<br>
+<a target="_blank" href="https://github.com/Open-Shell/Open-Shell-Menu"><img src="images/OpenShell.png" style="border: 0px solid ; width: 64px; height: 64px; float:left;" alt="Open-Shell website" title="Open-Shell website"></a><h1 style="padding-top: 12px; padding-bottom: 12px;">&nbsp; Open-Shell</h1>
+<h2>What is Open-Shell?</h2>
+<strong style="color: #2b88d8;">Open-Shell&#8482;</strong> is a collection of usability enhancements for Windows. It 
+has a customizable Start menu and Start button, adds a 
+toolbar for Windows Explorer, and supports a variety of smaller features.<br>
 <br>
-<h1>System Requirements</h1>
-Open-Shell works on Windows 7, Windows 8, Windows 8.1, Windows Server 2008 R2, 
-Windows Server 2012 and Windows Server 2012 R2. Both 32 and 64-bit versions are 
-supported (the same installer works for both). Some skins for the start menu 
-require Aero theme to be enabled. Others require at least Basic theme.<br>
+<h2>System Requirements</h2>
+Open-Shell works on Windows 7, Windows 8, Windows 8.1, Windows 10, and Windows 11, along with their server counterparts. Both 32 and 64-bit versions are 
+supported (the same installer works for both).<br>
 <br>
-<br>
-<h1>Components</h1><br>
-
+<h2>Components</h2>
 Open-Shell has three major components:<ul>
   <li><a href="Menu.html">Open-Shell Menu</a></li>
```

Comment:
`Open-Shell Menu` looks weird here (other components don't have `Open-Shell` in name).
I'd suggest to change it to `Start menu`.

Reply:
Or maybe `Classic Start Menu` (as other components have `Classic` too).

Reply:
I agree that it looks weird, but I personally think it makes more sense to keep it consistent with the product name (as shown as the titles of the other pages, etc.) Maybe it could be a future renaming effort?

Reply:
I will look at renaming the Menu.html component. Do you know of the other references off-hand?


> You can eventually remove them (in separate commit) to see if build and everything else goes well.

So I can remove the entire localization folder without issue?

Reply:
Oki, lets keep it as it is for now.